packages/currency/src/currencyManager.ts

```diff
@@ -246,7 +246,7 @@ export class CurrencyManager<TMeta = unknown> implements ICurrencyManager<TMeta>
       case RequestLogicTypes.CURRENCY.ERC777:
         if (NearChains.isChainSupported(currency.network)) {
           return isValidNearAddress(address, currency.network);
-        } else if (currency.network === 'tron') {
+        } else if (currency.network === 'tron' || currency.network === 'solana') {
```

Comment:
@leoslr Is it the fix or is the change of version of `multicoin-address-validator` also needed?

Reply:
Both are needed:
 - We need to tackle solana case here
 - The lib update was needed as well, version 5.2 didn't handle solana